### PR TITLE
Add 2023 Kansas Tax Rules

### LINKED ISSUE #4176: Add 2023 Kansas Tax Rules
2023 Income Tax Form Instructions: https://www.ksrevenue.gov/incomebook23.html

### PR DESCRIPTION
**Pull request by shuyu7zhan**
Fixes #4176

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ks/tax/income/rates/other.yaml [hunk 1]
```diff
@@ -22,5 +22,7 @@ metadata:
   reference:
   - title: 2022 Form K-40 instructions
     href: https://www.ksrevenue.gov/pdf/ip22.pdf
-  - title: Kansas 2022 Statute Article 32 §79-32,110_k
+    - title: 2023 Form K-40 instructions
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ks/tax/income/rates/other.yaml:25]: indentation

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: nothing changed in 2023?

File: policyengine_us/parameters/gov/states/ks/tax/income/credits/food_sales_tax/agi_limit.yaml [hunk 2]
```diff
@@ -11,5 +11,7 @@ metadata:
     href: https://www.ksrevenue.gov/pdf/ip21.pdf
   - title: 2022 Form K-40 instructions
     href: https://www.ksrevenue.gov/pdf/ip22.pdf
-  - title: Kansas 2021 Statute Article 32 §79-32,271
+  - title: 2023 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip23.pdf#page=5
+    - title: Kansas 2021 Statute Article 32 §79-32,271
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ks/tax/income/credits/food_sales_tax/agi_limit.yaml:16]: indentation

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: changelog_entry.yaml
Comment (shuyu7zhan) [reply changelog_entry.yaml:1]: > nothing changed in 2023?

No, all the same

**Comment by PavelMakarchuk** [comment]
No parameter changes in 2023

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ks/tax/income/agi/subtractions/oasdi/agi_limit.yaml [hunk 3]
```diff
@@ -11,6 +11,8 @@ metadata:
     href: https://www.ksrevenue.gov/pdf/ip21.pdf
   - title: 2022 Form K-40 instructions
     href: https://www.ksrevenue.gov/pdf/ip22.pdf
+  - title: 2023 Form K-40 instructions
+    href: https://www.ksrevenue.gov/pdf/ip23.pd#page=11
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ks/tax/income/agi/subtractions/oasdi/agi_limit.yaml:15]:
```suggestion
    href: https://www.ksrevenue.gov/pdf/ip23.pdf#page=11
```

**Comment by PavelMakarchuk** [comment]
> For posterity, the only things that changed in 2023 are programs we don't model:
> 
> > Personal Exemption for Disabled
> > Veterans - Beginning in tax year 2023,
> > any individual who has been honorably
> > discharged from active service in any
> > branch of the armed forces of the United
> > States and who is certified by the United
> > States department of veterans affairs or
> > its successor to be in receipt of disability
> > compensation at the 100% rate, if the
> > disability is permanent and was sustained
> > through military action or accident or
> > resulted from disease contracted while in
> > such active service, shall be allowed an
> > additional Kansas personal exemption of
> > $2,250.
> > New Tax Credits - Affordable Housing
> > Tax Credit (K-25), and Apprenticeship
> > Credit (K-24).
> > Apprenticeship Credit - Provides an
> > income tax credit for tax years commencing
> > after December 31, 2022 and ending before
> > January 1, 2026 for an eligible employer
> > who employs an apprentice pursuant to a
> > registered apprenticeship agreement.
> > Affordable Housing Tax Credit -
> > An income tax, privilege tax and premium
> > tax credit is available commencing
> > after December 31, 2022 for a qualified
> > development located in Kansas, as
> > determined by the Kansas H
> 
> Also thank you Kansas https://twitter.com/MaxGhenis/status/1772393902609256640

@shuyu7zhan could you take a look - I dont think we need a separate PR for the changes

**Pull request merged**